StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -97,10 +98,14 @@ final class UpdatePaymentMethodViewController: UIViewController {
         return button
     }()
 
+    private lazy var formFactory: SavedPaymentMethodFormFactory = {
+        let factory = SavedPaymentMethodFormFactory(viewModel: viewModel)
```

Comment:
I'm confused by the modeling here. UpdatePMVC owns a ViewModel and a Factory. The factory _also_ owns the ViewModel. How should we think about this?  Of the 3 entities here, who is responsible for what?

Reply:
All the factory needs it the paymentMethod.type from the ViewModel so that definitely can be scoped down in terms of owning the view model.

Reply:
It actually needs more than this, as sepa and us bank account need more. I will post a new revision shortly which turns the Factory to a class with one static function which creates PaymentMethodElements and has zero state.

Reply:
What's the purpose/responsibility of the ViewModel?  What belongs in there vs. UpdatePMVC?

Reply:
I've renamed ViewModel to a struct of Configuration -- they are read-only/computed values that are passed into the UpdatePaymentMethodViewController.  Any state can be managed by the ViewController directly.

One issue i ran into was with setting the initial error state -- i've introduced a callback in the factory, as I couldn't figure out an elegant way to enumerate through each text element of the form from the ViewController.